components/reusable-components/login-form/LoginForm.tsx

```diff
@@ -35,36 +33,31 @@ const LoginForm = ({ submitLoginForm }: LoginFormProps) => {
     password: Yup.string()
       .required('Задолжително внесете пасворд.')
       .min(8, 'Пасвордот би требало да содржи минимум 8 знаци.'),
-    // .matches(
-    //   /^(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]*$/,
-    //   'Мора да содржи еден број и еден посебен знак'
-    // ),
   });
-
   const handleSubmit = async (values: FormValues, { resetForm }: FormikHelpers<FormValues>) => {
     if (!turnstileToken) {
+      toast.error('Turnstile verification is required.');
       return;
     }
-
-    const formValues = {
-      email: values.email,
-      password: values.password,
-      cfTurnstileResponse: turnstileToken,
-    };
-
     try {
-      const response = await submitLoginForm(formValues);
-      if (response) {
-        toast.success(response);
-
+      const response = await login({
+        email: values.email,
+        password: values.password,
+        role: Role.content_manager,
```

Comment:
@Jasminamih Have you tested this?
As you can see in the API documentation(https://api.learnhub.mk/docs/#authentication-POSTcontent-login), the route we should call is `https://staging-api.learnhub.mk/content/login`. But in the `Role` enum we have defined content manager as: `content_manager = 'content_manager',` . Because of this the route you will call is `https://staging-api.learnhub.mk/content-manager/login`.  That route doesn't exist.